applications/nrf_desktop/README.rst

```diff
@@ -741,35 +872,58 @@ Page Down key
     * |nRF_Desktop_cancel_operation|
 
 Dongle
-~~~~~~
+++++++
 
 The following predefined buttons are assigned to peer control operations for the dongle:
 
 SW1 button
+    * The SW1 button is located on the top of the dongle.
+
+      .. figure:: /images/nRF52840_dongle_front_callouts.svg
```

Comment:
@pdunaj , same LED matter as for the keyboard and mouse. Let's keep the captions, but explain what the meaning of LEDs is.